org.blackfennec.app.extensions.core.yml

```diff
@@ -0,0 +1,18 @@
+id: org.blackfennec.app.extensions.core
+branch: '0.10'
```

Comment:
We discussed this briefly on Matrix, but I will repeat it here: you shouldn't use versioned branches for extensions targetting only apps. Once it's removed, I'm inclined to merge it, even though you will be switching back to the single manifest sooner than later.

Reply:
```suggestion
```